commands/history/inspect.go

```diff
@@ -309,9 +318,49 @@ func runInspect(ctx context.Context, dockerCli command.Cli, opts inspectOptions)
 		fmt.Fprintln(dockerCli.Out())
 	}
 
+	if rec.ExternalError != nil {
+		dt, err := content.ReadBlob(ctx, store, ociDesc(rec.ExternalError))
+		if err != nil {
+			return errors.Wrapf(err, "failed to read external error %s", rec.ExternalError.Digest)
+		}
+		var st spb.Status
+		if err := proto.Unmarshal(dt, &st); err != nil {
+			return errors.Wrapf(err, "failed to unmarshal external error %s", rec.ExternalError.Digest)
+		}
+		retErr := grpcerrors.FromGRPC(status.ErrorProto(&st))
+		for _, s := range errdefs.Sources(retErr) {
+			s.Print(dockerCli.Out())
+		}
+
+		var ve *errdefs.VertexError
+		if errors.As(retErr, &ve) {
+			dgst, err := digest.Parse(ve.Vertex.Digest)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse vertex digest %s", ve.Vertex.Digest)
+			}
+			name, logs, err := loadVertexLogs(ctx, c, rec.Ref, dgst, 16)
+			if err != nil {
+				return errors.Wrapf(err, "failed to load vertex logs %s", dgst)
+			}
+			if len(logs) > 0 {
+				fmt.Fprintf(dockerCli.Out(), "\n => %s:\n", name)
+				for _, l := range logs {
+					fmt.Fprintln(dockerCli.Out(), l)
+				}
+				fmt.Fprintln(dockerCli.Out())
+			}
```

Comment:
I would need to parse the output of inspect command for build summary in GHA. Could we have a prefix like `> ` that we use to print BuildKit config for `inspect` https://github.com/docker/buildx/pull/2684?